app/src/renderer/styles/app.css

```diff
@@ -122,12 +105,17 @@ input.tm-field {
   background: red;
   text-align: center;
   height: 1.5rem;
+  font-weight: 500;
   z-index: 1000000;
   width: 100%;
   top: 0;
   position: absolute;
 }
 
+#appzi-launch-button-cf1d801a-9653-4934-8a18-cea8c9e77495 {
```

Comment:
I don't think this works. are the button ids always the same?

Reply:
yep. it's based on our config.

/shrug